docs/docs/en/guide/howto/datasource-setting.md

```diff
@@ -51,16 +52,44 @@ mysql> GRANT ALL PRIVILEGES ON dolphinscheduler.* TO '{user}'@'localhost';
 mysql> FLUSH PRIVILEGES;
 ``` 
 
-Then, modify `./bin/env/dolphinscheduler_env.sh` to use mysql, change {user} and {password} to what you set in the previous step.
+For PostgreSQL: 
+```shell
+# Use psql-tools to login PostgreSQL
+psql
+# Create a database
+postgres=# CREATE DATABASE dolphinscheduler;
+# Replace {user} and {password} with your username and password
```

Comment:
I think we should use `<>` for the required parameter, could you please also change the other place?
```suggestion
# Replace <user> and <password> with your username and password
```

Reply:
OK, got it.

Reply:
Hi there, is it a bit strange for changing postresql part only? I think '{}' is clear for others, let's keep it or talk?